modules/preview-react/table/lib/TableDivider.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {BoxProps, Box} from '@workday/canvas-kit-react/layout';
+import {colors} from '@workday/canvas-kit-react/tokens';
+import {createComponent} from '@workday/canvas-kit-react/common';
+
+export const TableDivider = createComponent('hr')({
```

Comment:
Interesting, I wasn't aware you could have an `<hr />` directly inside a `<tbody>`. Doesn't seem like this is valid HTML based on the [MDN page for tbody](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody):

![image](https://user-images.githubusercontent.com/146020/230519775-76c5cf6e-03f3-46a0-a89d-2d18b8e7f469.png)

Reply:
@mannycarrera4 gave a little context on why you might have implemented the row dividers this way (to avoid having to write extra CSS to style the first and last rows differently). Let me know if you have any additional info on why an `<hr />` was used to implement the border between rows.

Trying to think about how I would implement the design from the spec (with the row borders not extending the full width of the container) and the only thing I'm coming up with is applying a padding to the `<tbody>` and styling the first and last rows differently to compensate.

Reply:
Yeah, I think the dividers might need to be removed altogether. 

The padding on `<tbody/>` doesn't work well. The issue is because when users want to implement a sticky left column, then the scrolling happens under that column and doesn't look great. 

So maybe, I will have to remove that divider altogether.